HW4_Diana_Mokhnitskaya/HW4_task2.py

```diff
@@ -0,0 +1,10 @@
+a = 2
+c = a * 2
+print("first variant of multiplication:" + str(c))
+a *= 2
+print("second variant of multiplication:" + str(a))
+
+d = c//2
+print("first variant of division:" + str(d))
+a //= 2
+print("second variant of division:" + str(a))
```

Comment:
the second variant should be byte operations. Please think how you can do this

Reply:
Fixed it.